test-it/results/EnumBasic.vtr

```diff
@@ -1,13 +1,13 @@
-2 jkl
```

Comment:
Looking at the EnumBasic.via file we do want the result of StringFormat primitive to output the enum string and not a number.

However for readJSON we want the number value to be returned when we read the terminal.